src/enums/api_url_enum.ts

```diff
@@ -38,6 +38,7 @@ export class ApiDeliveryUrlEnum {
     public static root_delivery_parcel = ApiDeliveryUrlEnum.e_root_delivery + "/parcel";
     public static endpoint_post_parcel_refuse = ApiDeliveryUrlEnum.root_delivery_parcel + "/refuse";
     public static endpoint_post_pending_hub_customer_handover = ApiDeliveryUrlEnum.root_delivery_parcel + "/pending_hub_customer_handover";
+    public static endpoint_post_defray_shopper = ApiDeliveryUrlEnum.e_root_delivery + "/delta_cost";
```

Comment:
Faudrais noté qqpart de migrer ce truc en mission Controller je trouve ca vraiment perturbant d avoir Delivery coté Js et Mission coté PHP. Si tu peux le noté qqpart